nixos/lib/make-options-doc/default.nix

```diff
@@ -108,9 +107,10 @@
   # instead of printing warnings for eg options with missing descriptions (which may be lost
   # by nix build unless -L is given), emit errors instead and fail the build
   warningsAreErrors ? true,
-}:
+}@args:
 
-let
+lib.warnIf (args ? documentType)
+(let
```

Comment:
This fails treefmt. Formatting will indent everything within the parentheses, so we should try and avoid adding them here.

One way to do so would be to abuse an `assert` statement:

```nix
assert lib.warnIf (args ? documentType) "nixosOptionsDoc: the argument `documentType` is deprecated and will be removed in a future release. It was a no-op with no behavior and can be safely removed." true;
```

That assertion will never throw (as it's always `true`), but it prints a warning if the condition fails.